src/main/java/uk/gov/companieshouse/presenter/account/validation/utils/StringValidator.java

```diff
@@ -0,0 +1,54 @@
+package uk.gov.companieshouse.presenter.account.validation.utils;
+
+import java.nio.charset.StandardCharsets;
+import java.util.Optional;
+import java.util.regex.Pattern;
+
+
+public final class StringValidator {
+
+    private static final int MAX_REGEX_LENGTH = 200;
+
+    private static final String VALID_CHARACTERS = "-,.:; 0-9A-Z&@$£¥€'\"«»?!/\\\\()\\[\\]{}<>*=#%+ÀÁÂÃÄÅĀĂĄÆǼÇĆĈĊČÞĎÐÈÉÊËĒĔĖĘĚĜĞĠĢĤĦÌÍÎÏĨĪĬĮİĴĶĹĻĽĿŁÑŃŅŇŊÒÓÔÕÖØŌŎŐǾŒŔŖŘŚŜŞŠŢŤŦÙÚÛÜŨŪŬŮŰŲŴẀẂẄỲÝŶŸŹŻŽa-zſƒǺàáâãäåāăąæǽçćĉċčþďðèéêëēĕėęěĝģğġĥħìíîïĩīĭįĵķĺļľŀłñńņňŋòóôõöøōŏőǿœŕŗřśŝşšţťŧùúûüũūŭůűųŵẁẃẅỳýŷÿźżž";
+
+    /**
+     * 
+     * @param unvalidatedString the raw string to be validated
+     * @param maxLength of a valid string
+     * @return Optional of string that passed validation or an empty optional if not.
+     * @throws IllegalArgumentException if an invalid max length is entered.
+     */
+    public static final Optional<String> validateString(final String unvalidatedString, final int maxLength) throws IllegalArgumentException {
+        boolean validLength = MAX_REGEX_LENGTH >= maxLength && maxLength > 0;
```

Comment:
This is very defensive coding. You're checking to make sure a developer in the future doesn't increase the max length of a field to above 200? I think we can assume that won't happen, and if it does the function will fail anyway.